sys-utils/lscpu-virt.c

```diff
@@ -500,6 +510,9 @@ static int is_vmware_platform(void)
 	if (sigaction(SIGSEGV, &oact, NULL))
 		err(EXIT_FAILURE, _("cannot restore signal handler"));
 
+	if (sigprocmask(SIG_SETMASK, &oset, NULL))
+		err(EXIT_FAILURE, _("cannot set signal mask"));
+
```

Comment:
The error messages of the two calls to sigprocmask are a bit inconsistent.

Reply:
Maybe "cannot restore signal mask" is better.

Reply:
Sounds good